myapp/app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,28 @@
+module SessionsHelper
+
+  def login(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
```

Comment:
- I use session method because it is easy to implement
- Using cookies method and hashing [like this](https://railstutorial.jp/chapters/advanced_login?version=5.1#cha-advanced_login) is too much for this training